radixdlt-core/radixdlt/src/test/java/com/radixdlt/sanitytestsuite/scenario/deserialization/DeserializationTestScenarioRunner.java

```diff
@@ -0,0 +1,243 @@
+/*
+ * (C) Copyright 2020 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+package com.radixdlt.sanitytestsuite.scenario.deserialization;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import com.radixdlt.DefaultSerialization;
+import com.radixdlt.atommodel.system.SystemParticle;
+import com.radixdlt.atommodel.tokens.FixedSupplyTokenDefinitionParticle;
+import com.radixdlt.atommodel.tokens.MutableSupplyTokenDefinitionParticle;
+import com.radixdlt.atommodel.tokens.TransferrableTokensParticle;
+import com.radixdlt.atommodel.tokens.UnallocatedTokensParticle;
+import com.radixdlt.atommodel.tokens.StakedTokensParticle;
+import com.radixdlt.atommodel.validators.RegisteredValidatorParticle;
+import com.radixdlt.atommodel.validators.UnregisteredValidatorParticle;
+import com.radixdlt.atomos.RRIParticle;
+import com.radixdlt.constraintmachine.Particle;
+import com.radixdlt.sanitytestsuite.scenario.SanityTestScenarioRunner;
+import com.radixdlt.sanitytestsuite.utility.ArgumentsExtractor;
+import com.radixdlt.serialization.DeserializeException;
+import com.radixdlt.serialization.Serialization;
+import org.bouncycastle.util.encoders.Hex;
+
+import java.util.Map;
+import java.util.function.BiConsumer;
+
+import static java.util.Optional.ofNullable;
+import static junit.framework.TestCase.assertNotNull;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public final class DeserializationTestScenarioRunner extends SanityTestScenarioRunner<DeserializationTestVector> {
+
+    private static final Serialization serialization = DefaultSerialization.getInstance();
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    private final ImmutableMap<String, BiConsumer<Map<String, Object>, Object>> assertEqualsMap;
+
+    public DeserializationTestScenarioRunner() {
+        Map<String, BiConsumer<Map<String, Object>, Object>> mutableMap = Maps.newHashMap();
```

Comment:
Alternative version might be `var mutableMap = Maps.<String, BiConsumer<Map<String, Object>, Object>>newHashMap();`.
No big difference, just little bit more consistent with modern java style.